src/Microsoft.ML.StandardLearners/Standard/MultiClass/PairwiseCouplingTrainer.cs

```diff
@@ -167,7 +168,7 @@ private IDataView MapLabels(RoleMappedData data, int cls1, int cls2)
                 return MapLabelsCore(NumberDataViewType.Double, (in double val) => val == key1 || val == key2, data);
             }
 
-            throw Host.ExceptNotSupp($"Label column type is not supported by PKPD: {lab.Type}");
+            throw Host.ExceptNotSupp($"Label column type is not supported by PkPdTrainer: {lab.Type.RawType}");
```

Comment:
>PkPdTrainer [](start = 77, length = 11)

Can you rename this too?

Reply:
Thanks!

---
In reply to: [260864060](https://github.com/dotnet/machinelearning/pull/2749#discussion_r260864060) [](ancestors = 260864060)